src/components/Dissolution/CompleteResolution.vue

```diff
@@ -125,7 +125,7 @@
               </label>
             </v-col>
             <v-col cols="12" sm="9">
-              <date-picker
+              <DatePickerShared
```

Comment:
This is just cleanup (and consistency with what I did in Edit UI to clearly label external/shared components).

Reply:
Also, the CamelCase format allows you to F12 on it to go to its declaration (which the kebab-case format doesn't support).